laravel/app/Http/Controllers/ScheduleDataController.php

```diff
@@ -469,306 +380,156 @@ public function updateMilestones($id){
-		          ->where('option',$choice)
-		          ->count() != 0)
-		  {
-			// generate notification and delete data
-			$nofity = true;
-			$overwrittenChoices[2] = 3;
-            // If user enters 3rd choice, update option 3 data; otherwise, make old 3nd choice data invalid.
-            if(!is_null($schedule_data3)){
-                Option::where('date', $date)
-                        ->where('resident', $resident)
-                        ->where('option',$choice)
-                        ->update([
-                            'schedule' => $split[2],
+                } else {
+                    $overwrittenChoices[$i] = 0;
+                    // Insert data
+                    if (!is_null($schedule_data[$i])) {
+                        // Insert data
+                        Option::insert([
+                            'date' => $date,
+                            'resident' => $resident,
+                            'schedule' => $split[$i],
                             'attending' => $attending,
-                            'milestones'=>$_REQUEST['milestones3'],
-                            'objectives'=>$_REQUEST['objectives3'],
-                            'anesthesiologist_id'=>$pref_anest3,
-                            'isValid'=>1
+                            'option' => $choice,
+                            'milestones' => $_REQUEST['milestones' . $choice],
+                            'objectives' => $_REQUEST['objectives' . $choice],
+                            'anesthesiologist_id' => $pref_anest[$i],
+                            'isValid' => 1
                         ]);
-            } else {
-                Option::where('date', $date)
-                  ->where('resident', $resident)
-                  ->where('option',$choice)
-                  ->delete();
-                  // ->update(['isValid'=>0]);
-              }
-		  }
-		else {
-			$overwrittenChoices[2] = 0;
-            if(!is_null($schedule_data3)){
-                // Insert data
-                Option::insert([
-                    'date' => $date,
-                    'resident' => $resident,
-                    'schedule' => $split[2],
-                    'attending' => $attending,
-                    'option' => $choice,
-                    'milestones'=>$_REQUEST['milestones3'],
-                    'objectives'=>$_REQUEST['objectives3'],
-                    'anesthesiologist_id'=>$pref_anest3,
-                    'isValid'=>1]
-                );
+                    }
+                }
             }
-		}
-
+        }
 
-		// data was overwritten, send a notification
-		if($notify == true){
-			// please make sure to change the email here
-			self::notifyResidentOverwrittenPreferences('', $_SERVER["HTTP_EMAIL"], $residentName, $date, $overwrittenChoices);
-		}
+        // data was overwritten, send a notification
+        if ($notify == true) {
+            // please make sure to change the email here
```

Comment:
What does this comment mean? I'm not sure we should be giving `notifyResidentOverwrittenPreferences` an empty string.